src/APIFunctions/3DPrinting.js

```diff
@@ -15,7 +23,7 @@ import axios from 'axios'
  * @returns {boolean} - If the request was successfully submitted or not.
  */
 export async function submit3DPrintRequest (printRequest) {
-  let isSubmitted = false
+  const isSubmitted = false
```

Comment:
this should return the class
```suggestion
  const isSubmitted = new ApiResponse()
```